src/azure-cli/HISTORY.rst

```diff
@@ -73,6 +73,7 @@ Release History
 
 **NetAppFiles**
 
+* Update for RP R4.5/API version 2019-06-01 : for create/update volume, added new argument --protocol-types
```

Comment:
"Update for RP R4.5/API version 2019-06-01 :" This is all still extraneous to customers. The format I gave you is quite literally what the standard release notes would look like for this.